src/views/ItemView.js

```diff
@@ -1270,6 +1286,7 @@ ItemView.propTypes = {
     staffMembers: PropTypes.object,
     servicePoints: PropTypes.object,
     openLoans: PropTypes.object,
+    tagSettings: PropTypes.shape({ records: PropTypes.arrayOf(PropTypes.object) }),
```

Comment:
IMHO, it makes not too much sense in such shapes, since you don't specify all props. Usually we put proptypes.obj